src/System Application/App/File System/src/Account/FileAccounts.Page.al

```diff
@@ -0,0 +1,322 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.FileSystem;
+
+using System.Telemetry;
+
+/// <summary>
+/// Lists all of the registered file accounts
+/// </summary>
+page 9450 "File Accounts"
+{
+    PageType = List;
+    Caption = 'File Accounts';
+    ApplicationArea = All;
+    UsageCategory = Administration;
+    SourceTable = "File Account";
+    SourceTableTemporary = true;
```

Comment:
Source table is temporary by default

Reply:
Same in the email module:
https://github.com/microsoft/BCApps/blob/4c7b6487cb55b1b750997f3ef33b1a92f8e678b8/src/System%20Application/App/Email/src/Account/EmailAccounts.Page.al#L20